modules/ROOT/pages/clauses/match.adoc

```diff
@@ -490,3 +490,135 @@ The above query uses the xref:functions/aggregating.adoc#functions-collect[`coll
 
 For more information about how Cypher queries work, see xref:clauses/clause-composition.adoc[].
 
+[role=label--new-5.26]
+[[dynamic-match]]
+== MATCH nodes and relationships using dynamic node labels and relationship types
+
+Node labels and relationship types can be referenced dynamically in parameters and variables when matching nodes and relationships.
+
+.Syntax for matching node labels dynamically
+[source, syntax]
+----
+MATCH (n:$(<expr>))
+MATCH (n:$any(<expr>))
+MATCH (n:$all(<expr>))
+----
+
+.Syntax for matching relationship types dynamically
+[source, syntax]
+----
+MATCH ()-[r:$(<expr>))]->()
+MATCH ()-[r:$any(<expr>)]->()
+MATCH ()-[r:$all(<expr>))]->()
+----
+
+The expression must evaluate to a `STRING NOT NULL | LIST<STRING NOT NULL> NOT NULL` value.
+
+.Match labels dynamically using the `all()` function
+[source, cypher]
+----
+WITH "Movie" AS label
+MATCH (movie:$all(label))
+RETURN movie AS movieNodes
```

Comment:
I think if we are showcasing all explicitly then we should have more than one label and explain it is the conjunction